doc/content/verification/ver-1a.md

```diff
@@ -49,7 +49,7 @@ with
     \phi = \frac{\text{source concentration}}{\text{layer concentration}} = \frac{1}{S k_b T},
 \end{equation}
 
-where the layer concentration is that at the interface with the source ($\phi$ is constant in time), $k_b$ is the Boltzmann constant as defined in [PhysicalConstants](/source/utils/PhysicalConstants.md), and the $\alpha_n$ are the roots of
+where the layer concentration is that at the interface with the source ($\phi$ is constant in time), $k_b$ is the Boltzmann constant as defined in [PhysicalConstants](source/utils/PhysicalConstants.md), and the $\alpha_n$ are the roots of
```

Comment:
This sentence structure needs some reworking.

Reply:
```suggestion
where the "layer concentration" is the concentration quantity at the interface with the source ($\phi$ is constant in time), $k_b$ is the Boltzmann constant as defined in [PhysicalConstants](source/utils/PhysicalConstants.md), and the $\alpha_n$ are the roots of
```

Maybe this?